src/ncp/posix/netif.cpp

```diff
@@ -202,6 +202,23 @@ void Netif::SetNetifState(bool aState)
     }
 }
 
+void Netif::Ip6Receive(const uint8_t *aBuf, uint16_t aLen)
+{
+    otbrError error = OTBR_ERROR_NONE;
+
+    VerifyOrExit(aLen <= kIp6Mtu, error = OTBR_ERROR_DROPPED);
+    VerifyOrExit(mTunFd > 0, error = OTBR_ERROR_INVALID_ARGS);
```

Comment:
How about `INVALID_STATE` instead of `INVALID_ARGS`? since `mTunFd` is not an arg?